library/src/main/java/com/dslplatform/json/DslJson.java

```diff
@@ -2786,6 +2786,20 @@ public boolean serialize(final JsonWriter writer, final Type manifest, @Nullable
 	 * @throws IOException error when unable to serialize instance
 	 */
 	public final void serialize(@Nullable final Object value, final OutputStream stream) throws IOException {
+	    serialize(value, null, stream);
+    }
+
+	/**
+	 * Convenient serialize API.
+	 * In most cases JSON is serialized into target `OutputStream`.
+	 * This method will reuse thread local instance of `JsonWriter` and serialize JSON into it.
+	 *
+	 * @param value    		instance to serialize
+	 * @param typeManifest 	type manifest
+	 * @param stream 		where to write resulting JSON
+	 * @throws IOException error when unable to serialize instance
+	 */
+	public final void serialize(@Nullable final Object value, @Nullable Class<?> typeManifest, final OutputStream stream) throws IOException {
```

Comment:
If we are gonna have such method, it should accept a Type instead of Class and it should not be nullable

I guess the method is useful (even if there are already too many methods here...)

I kind of am not a fan of such method (since you can send in a different object from the signature),
but it is useful.
What might make more sense it to have a generic signature if we are passing in a Class

Reply:
I think we can avoid this method. Also I found a bug (?) if deserialize a child class the resulting json will be without discriminator.